tests/pkiunittest.py

```diff
@@ -179,7 +179,7 @@ def user_revoke(self, *opts):
         if not self.reqid and '--help' not in opts:
             raise CertFileError('Could not revoke cert due to missing request ID\n')
         args = opts + ('--test', self.reqid, 'osg-pki-tools unit test - user revoke')
-        cmd = (os.path.join(SCRIPTS_PATH, 'osg-user-cert-revoke'),) + args
+        cmd = ('/bin/sh', os.path.join(SCRIPTS_PATH, 'osg-user-cert-revoke'),) + args
```

Comment:
Ah, it looks like at least the rpm takes care of marking them executable (the ones installed under `/usr/bin`).  Anyway, it should work as you have it.

Maybe for future consideration... My thought is actually that any scripts you intend to invoke that start with a `#!` line really ought to be executable and invoked directly (rather than explicitly passing it to the interpreter, in this case `/bin/sh`).

(Interestingly, for the scripts in question here, `osg-user-cert-revoke` is shell, but it invokes `osg-cert-revoke` directly, which is python --but not executable-- in the sources.  The rpm makes it executable, but for non-rpm installs my guess is this will fail with permission denied.)